t/lib/t/MusicBrainz/Server/Controller/WS/2/DefaultFormat.pm

```diff
@@ -0,0 +1,91 @@
+package t::MusicBrainz::Server::Controller::WS::2::DefaultFormat;
```

Comment:
Pretty sure most of the test packages map directly to the package name they're testing, so `t/lib/t/MusicBrainz/Server/WebService/Format.pm` seems like a better location.